packages/fiori/src/IllustratedMessage.js

```diff
@@ -106,14 +109,28 @@ const metadata = {
  * @class
  *
  * <h3 class="comment-api-title">Overview</h3>
- * An <code>ui5-illustrated-message</code> recommended combination of a solution-oriented message, an engaging
+ * An <code>ui5-illustrated-message</code> is a recommended combination of a solution-oriented message, an engaging
  * illustration, and conversational tone to better communicate an empty or a success state than just show
  * a message alone.
  *
  * Each illistration has default internationalised title and subtitle texts. Also they can be managed with
  * <code>titleText</code> and <code>subtitleText</code> properties.
  *
- * <b>Note:</b> By default BeforeSearch illustration is loaded.
+ * То display the desired illustration, use the <code>name</code> property, where you can find the list of all available illustrations.
+ * <br><br>
+ * <b>Note:</b> By default the “BeforeSearch” illustration is loaded. To use other illustrations, make sure you import them in addition:
+ * <br>
+ * <code>import @ui5/webcomponents-fiori/dist/illustrations/NoData.js</code>
+ *
+ * <h3>Structure</h3>
+ * The IllustratedMessage consists of the following elements, which are displayed below each other in the following order:
```

Comment:
If you use IllustratedMessage here, then it's better to also change the very first word of the Overview from "ui5-illustrated-message" to "IllustratedMessage".

`An <code>IllustratedMessage</code> is a recommended...`